core/templates/pages/splash-page/splash-page.component.html

```diff
@@ -1,201 +1,727 @@
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Space')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Algebra')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Music')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Chemistry')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Poetry')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Puzzles')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Calculus')]>" class="oppia-splash-background-icon" alt="">
-
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Business')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Geography')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Biology')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Genetics')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Space')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Algebra')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Music')]>" class="oppia-splash-background-icon" alt="">
-        <img ng-src="<[$ctrl.getStaticSubjectImageUrl('Chemistry')]>" class="oppia-splash-background-icon" alt="">
+      <div class="oppia-splash-button-container">
+        <form action="<[$ctrl.classroomUrl]>">
+          <button type="submit" ng-click="$ctrl.onClickBrowseLessonsButton()"
+                  class="btn oppia-splash-button oppia-splash-button-browse oppia-splash-browse-button-position oppia-transition-200"
+                  translate="I18N_ACTION_BROWSE_LESSONS">
+          </button>
+        </form>
       </div>
+    </div>
+    <div class="oppia-splash-section-one-right">
+      <picture class="oppia-splash-main-image image-desktop">
+        <source type="image/webp" ng-srcset="<[$ctrl.getStaticImageUrl('/splash/splashMainDesktop.webp')]>">
+        <source type="image/jpeg" ng-srcset="<[$ctrl.getStaticImageUrl('/splash/splashMainDesktop.png')]>">
+        <img ng-src="<[$ctrl.getStaticImageUrl('/splash/splashMainDesktop.png')]>" alt="">
+      </picture>
+      <picture class="oppia-splash-main-image image-mobile">
+        <source type="image/webp" ng-srcset="<[$ctrl.getStaticImageUrl('/splash/splashMainMobile.webp')]>">
+        <source type="image/jpeg" ng-srcset="<[$ctrl.getStaticImageUrl('/splash/splashMainMobile.png')]>">
+        <img ng-src="<[$ctrl.getStaticImageUrl('/splash/splashMainMobile.png')]>" alt="">
+      </picture>
```

Comment:
This should work too and should be more efficient (in terms of not downloading unneeded images).
```suggestion
      <picture class="oppia-splash-main-image">
        <source type="image/webp" media="(min-width: 769px)" ng-srcset="<[$ctrl.getStaticImageUrl('/splash/splashMainDesktop.webp')]>">
        <source type="image/jpeg" media="(min-width: 769px)" ng-srcset="<[$ctrl.getStaticImageUrl('/splash/splashMainDesktop.png')]>">
        <img ng-src="<[$ctrl.getStaticImageUrl('/splash/splashMainDesktop.png')]>" alt="">

        <source type="image/webp" media="(max-width: 768px)" ng-srcset="<[$ctrl.getStaticImageUrl('/splash/splashMainMobile.webp')]>">
        <source type="image/jpeg" media="(max-width: 768px)" ng-srcset="<[$ctrl.getStaticImageUrl('/splash/splashMainMobile.png')]>">
        <img ng-src="<[$ctrl.getStaticImageUrl('/splash/splashMainMobile.png')]>" alt="">
      </picture>
```

Reply:
I tried this, but it showed both images on both desktop and mobile, so reverted this back.

Reply:
Oh, I edited it now, but feel free to include it in follow-up PR?